frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -307,7 +307,7 @@ sidebar.home = Home
 sidebar.dashboard = Overview
 sidebar.analytics = Analytics
 sidebar.analytics.users = User Analytics
-sidebar.analytics.user-loyalty = User loyalty
+sidebar.analytics.visitor-loyalty = Visitor Loyalty
```

Comment:
We use both of them; "Visitor" for web apps and "User" for mobile apps. This is probably one of those cases. We should double check this with Duygu.

Reply:
Yes, actually she confirmed that for non-web users it will be "User". I will keep both of them.